API_Auto/src/test/java/APIAssignments/BaseUrl.java

```diff
@@ -0,0 +1,10 @@
+package APIAssignments;
+
+public class BaseUrl {
+
+    public static String getUrl() {
+        String url = "https://automationexercise.com/api/productsList";
+        return url;
```

Comment:
Please try using .properties file for URL

Reply:
Done